tests/e2e/test-site/index.htm

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
What's the reason for using `index.htm` and `index.html`?

Reply:
Ha ha, this was going to be the first comment I left when I came back to this. It's a trick to be able to run the E2E tests in two _nearly_ identical ways with the most amount of code reuse. [The `htm` file is for testing with the live AM server](https://github.com/ForgeRock/forgerock-javascript-sdk/pull/19/files/04db4c3c0ee6fc3dcf83ac52f797d444c9b01f26#r356708333) and `html` is for the mocked version. Notice the lack of `<script src="server.js"></script>` in this file.

The only configuration option I could find in `http-server` to allow for "run this for X and this for Y" was the file extension option. I don't like how "hacky" it feels, but it was the least disruptive approach I could find.

Reply:
Ah, got it.  Yeah, I wish there was a way to either point at a different folder and still pull in the same externally referenced files, or use some criteria (querystring, hostname, etc) to dynamically add server.js.  I poked around for a couple minutes and didn't see a good alternative, either.